src/vuex/modules/session.js

```diff
@@ -239,6 +239,16 @@ export default ({ apollo }) => {
     setPreferredCurrency({ state, dispatch }, currency) {
       state.preferredCurrency = currency
       dispatch(`storeLocalPreferences`)
+    },
+    async getPreferredCurrency() {
+      const localPreferences = localStorage.getItem(USER_PREFERENCES_KEY)
```

Comment:
why do we need to load this here again? shouldn't that already loaded? preferredCurrency is in the state of this module

Reply:
yes, but every time you restart the app the state empties again. On app initialization `state.preferredCurrency` equals `""`. Persistence is coming from `localStorage` actually

Reply:
at some point on app start `loadLocalPreferences` should be called right?

Reply:
it is in `initializeApp`. It was überflüssig there, extra, I don't know how to say

Reply:
when the app loads it should load the preferences. no need to load the preference again at this point is what I am saying. At this point the preferences should have been loaded an the preferred currency should be available in the sate.

Reply:
Yeah, you were right. There was a return preventing this to work on develop because of analytics, but it was extra since the `setAnalyticsCollection` and `setErrorCollection` have their own returns each